windows.md

```diff
@@ -390,6 +390,41 @@ With those compatibility things out of the way, you're ready to start the system
     4. Open the start menu and search for "Ubuntu". Start it - it should ask you to create a user with a password. This will be your user to log in to your Ubuntu Linux Subsystem - note down the username and password somewhere secure to make sure you do not forget it.
     5. Open the start menu and search for "Docker Desktop". Start it and go to the Settings. Under the General tab, you will find an option called "Use WSL 2 based engine". Make sure this is checked.
 
+    When opening Docker Desktop during Option A or Option B, an error message `Docker Desktop - Windows Hypervisor is not present` may appear:<br><br>
+    
+    <img src="image here"><br><br>
+
+    If this appears for you, follow the next steps to enable virtualization on your machine (if you don't receive the error, continue with the step further below to test Docker):
+
+    Check if your machine supports virtualization by running the following command on powershell as an administrator:
```

Comment:
check for casing of project names

(in the full changes of the whole PR)

Reply:
I had only the PowerShell written wrong
566bd9e0543d9f18bfe43cfb34bf540be299aa90